docs/migration/datadog/vector.md

```diff
@@ -0,0 +1,165 @@
+# Vector
+
+*Migrating from Datadog to Siglens using Vector*
+
+## 1. Install Vector and Configure Datadog Agent
+
+Prerequisites: 
+- Datadog Agent (You can follow [this guide](https://docs.datadoghq.com/getting_started/agent/) to install the agent based on your OS)
+- Vector (You can follow [this guide](../../log-ingestion/vector.md#1-installation) to get started with installing vector)
+
+### Configure Datadog Agent
+
+1. Navigate to the Datadog Agent's configuration file. By default, it's located at `/opt/datadog-agent/etc/datadog.yaml`.
```

Comment:
Is this the default location for all operating systems. If not you should specify where they should look based on their operating system, or link to data dog documentation if they have some about where this file should be

Same comment for the path specified on line 70